.changeset/bloated-ghosts-expand.md

```diff
@@ -0,0 +1,5 @@
+---
+"@fluidframework/aqueduct": major
+---
+
+Return `IFluidDataStoreChannel` interface from `PureDataObjectFactory.instantiateDataStore` instead of `FluidDataStoreRuntime` implementation class.
```

Comment:
Nit: a bit more focused on consumers of aqueduct as the readers of the release notes.
```suggestion
The return type of `PureDataObjectFactory.instantiateDataStore` is changing from `FluidDataStoreRuntime` to `IFluidDataStoreChannel`.
```